ArduCopter/AP_Arming.cpp

```diff
@@ -218,19 +224,6 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
         }
 
         #if FRAME_CONFIG == HELI_FRAME
-        if (copter.g2.frame_class.get() != AP_Motors::MOTOR_FRAME_HELI_QUAD &&
```

Comment:
There is a check of this same thing for multirotor to say if the heli was selected in the #else statement for this #if statement. Why delete this statement and not the same statement in the #else.